docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,166 @@
+
+![](images/ts-8.png)
+
+Click **'Add rule'** to save the rule, or click the **'Cancel'** button to discard changes.
+
+After configuring the rules, click **'Save and update rules'** to confirm the changes or click **'Close'** to discard them.
+
+You can edit or remove rules in the tracking detail view's rule list.
+
+![](images/ts-9.png)
+
+### Entities
+
+In this section, you can define which [Entities](/docs/understanding-tracking-design/understanding-events-entities/index.md) should be tracked as context for this event.
+
+To add multiple entities, click the **'Add entities'** button. A new dialog box will appear to select entities. This includes both entities from Iglu Central and your own custom entities.
+
+![](images/ts-10.png)
+
+To view detailed information about the properties of an entity, click **'View properties'**. You can use the search bar to find an entity by name.
+
+Once you have selected the entities you want to add to the tracking scenario, click the **'Select entities'** button to confirm, or the **'Cancel'** button to discard your selections. On the next screen, you can configure additional rules. For example, you can specify how many instances of an entity should be tracked with this event, or whether the entity is optional or should always be tracked.
+
+![](images/ts-11.png)
+
+Click **'Save entities'** to save your changes, or **'Back'** to discard them.
+
+After saving the entities, you will see a list of them in the detail view, where you can edit, add, or remove them.
+
+![](images/ts-12.png)
+
+## Publishing a Tracking Scenario
+
+Once you have finalized your definition of the tracking scenario and completed all necessary approval stages, you can publish it by clicking the **'Publish scenario'** button located at the right side of the scenario name in the detailed view.
+
+![](images/ts-13.png)
+
+Publishing a new scenario indicates an intention to implement the tracking defined in it. For data consumers, a published scenario signals that the tracking associated with the event complies with the rules contained in it.
+
+Once published, the scenario's status will appear as "Published" in the list view.
+
+If you need to edit a published scenario, you can do so by selecting the "Edit" button. This action will reset the scenario's status to "Draft."
+
+![](images/ts-14.png)
+
+## Sharing a Tracking Scenario
+
+If you want to share a tracking scenario, whether it is published or still a draft, follow these steps. First, go to the Tracking scenario tab. Then, click on the name of the tracking scenario you want to share. From the detailed view, click the **'Share scenario'** button, which is located near the publish button.
+
+![](images/ts-15.png)
+
+A dialog box will appear with a link and a **'Copy'** button.
+
+![](images/ts-16.png)
```

Comment:
Can we update the image with one that doesn’t have the url of our staging environment in it? 
Also we need to update the header of this dialog to Tracking Scenario, rather than Implementation Scenario

Reply:
I'm able to edit HTML to replace `implementation scenario` for `tracking scenario` but can't edit the link url. It would be great to get some help @jamessnowplow and @DianaIgnatescu

Reply:
We should change the UI - I will create a Jira bug ticket

Reply:
Updated